src/data/uiConfig.ts

```diff
@@ -306,3 +306,10 @@ export const isWithdrawTokenPriceEnabled = (config: ConfigProps) => {
     config.cellarNameKey === CellarNameKey.REAL_YIELD_USD
   )
 }
+
+export const isUseBigBacktestingModal = (config: ConfigProps) => {
```

Comment:
not a blocker, after release rename this to `backtestingModalWidth.
Because this is not a boolean value, it's not represent the name of the function.

Reply:
noted